definitions.md

```diff
@@ -0,0 +1,62 @@
+# Definitions
+
+## Application Server
+
+This is the server that hosts the application.
+
+## End User Application
+
+This is the application used by the end user to connect to the application server and to interact with it.
+
+## Message
+
+This is the content that the application server is sending to the end user application.
+
+## Push System
```

Comment:
maybe we could more clearly say that this is not the only system but the one that is used when the application would like to receive a message even when not currently running